### PR TITLE
add databroker

### PR DESCRIPTION
**Pull request by CJ-Wright**

### CODE REVIEW

**Comment by CJ-Wright** [comment]
Pending https://github.com/conda-forge/pymongo-feedstock/pull/2

File: recipes/databroker/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    - tzlocal
+    - ujson
+
+test:
+  imports:
+    - databroker
+
+about:
+  home: https://github.com/NSLS-II/databroker
+  license: BSD
```
Comment (synapticarbors) [inline recipes/databroker/meta.yaml]: `license: BSD-3-Clause`

File: recipes/databroker/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    - requests
+    - singledispatch  # [py27]
+    - six
+    - tifffile
+    - tornado
+    - tzlocal
+    - ujson
+
+test:
+  imports:
```
Comment (synapticarbors) [inline recipes/databroker/meta.yaml]: You might consider testing the imports of the various submodules.

**Comment by synapticarbors** [comment]
It looks like `mongoquery` is not available for py36 on win-64. There may be other packages that are causing issues, but the error message is certainly unclear. I just went through and did some quick `conda search` on the packages, but not exhaustively.

**Comment by synapticarbors** [comment]
oops. Didn't see your mention of the issue before.

**Comment by CJ-Wright** [comment]
Pending https://github.com/conda-forge/mongoquery-feedstock/pull/4

**Comment by CJ-Wright** [comment]
@synapticarbors when you get a chance can you take a look at the broken circle build. I have no idea what is going on, I can build it locally no problem.

**Comment by CJ-Wright** [comment]
@conda-forge/staged-recipes could I please get a review/merge? (This is a noarch package so we don't need to wait on the OSX build)

**Comment by mwcraig** [comment]
Looks good, merging!

**Comment by CJ-Wright** [comment]
@mwcraig Thank you!

**Pull request merged**